aws/cognito/lambda.tf

```diff
@@ -124,7 +124,7 @@ resource "aws_lambda_function" "create_auth_challenge" {
 }
 
 resource "aws_lambda_layer_version" "create_auth_challenge_nodejs" {
-  filename            = "/tmp/verify_create_auth_challenge_nodejs.zip"
+  filename            = "/tmp/create_auth_challenge_nodejs.zip"
```

Comment:
No way I missed it... I was trying to find discrepancies in file names because you had probably been doing quite a few copy and paste moves.